### DISCUSSION TITLE
[Feature] Sub album within album

### DISCUSSION THREAD

**Discussion by SGPxyz** [immich-app/immich#3065] [Feature Request]
### The feature

Hello everyone,

Firstly, i would like to thanks everybody who are working on immich. It's a really great app that i use on a daily basis, and a real alternative to GAFAMs classic photos gallery tools.

I would like to know if is it planned, in the roadmap, to set up the possibility to create "sub album" within albums (like sub folder within folder on a server/computer)?

Best regards,

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
There is an open feature request for this: #2073. There is no roadmap per se, and so there are no concrete plans to add this at the moment.

**Reply by SGPxyz** [reply]
Thank you for your answer  :)!
